src/actions/selected.js

```diff
@@ -75,18 +70,12 @@ export const acClearSelectedItemActiveTypes = () => ({
 export const tSetSelectedDashboardById = id => async (dispatch, getState) => {
     dispatch(acSetSelectedIsLoading(true))
 
-    const snackbarTimeout = setTimeout(() => {
-        const dashboardName = (
-            sGetDashboardById(getState(), id) || EMPTY_DASHBOARD
-        ).displayName
-        if (sGetSelectedIsLoading(getState()) && dashboardName) {
-            loadingDashboardMsg.name = dashboardName
+    const alertTimeout = setTimeout(() => {
+        const name = sGetDashboardById(getState(), id)?.displayName
 
+        if (sGetSelectedIsLoading(getState()) && name) {
             dispatch(
-                acReceivedSnackbarMessage({
-                    message: loadingDashboardMsg,
-                    open: true,
-                })
+                acSetAlertMessage(`${i18n.t('Loading dashboard')}: ${name}`)
```

Comment:
The string should probably have a placeholder for the name.
Some languages might need to position the name before the rest of the text, also, the `: ` can cause problems (ie. RTL)

Reply:
Then we should find another separator. I can't put ":" into the translation string bc it is used as a namespace separator

Reply:
True. But regardless of the `:` problem, the interpolation with the placeholder still applies.

Reply:
@edoardo updated to use interpolation